### PR TITLE
Added a binary mod to the URI.open method

### PR DESCRIPTION
**Pull request by askonev**

### CODE REVIEW

**Comment by ShockwaveNN** [comment]
Again - why this needed

File: helpers/image_helper.rb [hunk 1]
```diff
@@ -6,11 +6,14 @@
...
+  # The method returns the file size taking url
+  # @param url [String] url image
+  # @return [Integer] size image
   def self.get_image_size(url)
-    Tempfile.create('images-') do |fo|
-      fo.binmode
-      fo.write(URI.parse(url).open(&:read))
-      File.size(fo.path)
+    Tempfile.create('images-') do |tmpfile|
+      binary_image = URI.open(URI.parse(url), 'r:binary').read
```
Comment (ShockwaveNN) [inline helpers/image_helper.rb]: `Uri.open(url).read` will not close connection

https://stackoverflow.com/a/4217806/820501

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]

**Comment by askonev** [comment]
Why, when using the open method, activating the binmod?
 
If you work with an image without a binmode, then part of the data will be lost.

Why the construction is not recommended:

`URI.parse (url) .open ('r: binary') {| t | t.read}`

explained in this article:

https://stackoverflow.com/questions/17633293/ruby-binary-string-to-io

Why specifically the force-encoding method:

https://stackoverflow.com/questions/21614228/what-is-the-difference-between-encode-and-force-encoding-in-ruby

**Comment by ShockwaveNN** [comment]
I think it's better not go into deep technical stuff, but just attach error or incorrect result which fix this PR

**Review by ShockwaveNN — APPROVED** [review]

**Review by flaminestone — APPROVED** [review]

**Pull request merged**